content/basics/ably.textile

```diff
@@ -0,0 +1,90 @@
+---
+title: What is Ably?
+meta_description: "A brief introduction to Ably and its features."
+---
+
+Ably’s realtime platform provides  a global realtime pub/sub network, that is scalable and fault-tolerant through our Realtime or REST interfaces. Built on the "Four Pillars of Dependability":#infrastructure Ably provides reliable performance available when needed.
+
+h2(#what). What can you build with Ably?
+
+Ably is used across a variety of industries to provide scalable building realtime collaborative web experiences, such as:
+
+* Live update services
+* Chat applications
+* Virtual events
+* Multi-user virtual spaces and applications
+* Internet of Things
+* And much more
+
+The following sections will cover some of the different APIs and features Ably provides which will give you a deeper insight into what can be done with Ably. There's also the "How to use Ably topic":/basics/use-ably which will go over the basics of connecting with Ably.
+
+Alternatively take a look at our "Use cases and solutions":https://ably.com/solutions topic for specific example use cases.
```

Comment:
Instead of "Alternatively," 

We should say something like..."To learn more about how to use Ably, check out our use cases and solutions."

Reply:
Do we like "check out" I'm also not opposed to "look at"